pkg/test/framework/featurelabels/features.go

```diff
@@ -0,0 +1,18 @@
+package featurelabels
```

Comment:
nit: just "features" ... drop use of "label" everywhere, since it will be confusing to have "labels" and "featurelabels".